docs/introduction/framework-extensibility.md

```diff
@@ -13,22 +13,64 @@ as well as a list of customizations that are not (and will not be) possible at a
+* Adding custom javascript into administration
+* [Adding form option into existing form](https://github.com/shopsys/demoshop/commit/898d111879aef40196f79ac763373560f44aef59#diff-3293b000b06ad6c0280341584c4d661d)
+* [Extending administration theme](https://github.com/shopsys/demoshop/commit/d0e0eaaa2eeac5e1c90d8a29be5c827c4a067b9f)
+* [Changing an entity association](https://github.com/shopsys/demoshop/commit/9931083ea37ad611568e32bc1a9c8cf203401809)
+    * this change is complicated and potentially dangerous
 
 ## Which issues are going to be addressed soon
 * Extending data fixtures (including performance data fixtures)
 * Extending data grids in the administration
 * Extending all forms in the administration without the need of the template overriding
-* Hiding the existing features and functionality
+* Extending classes like Repositories don't lead to change of project-base tests
+* Incorrectly extended entities cause clear exceptions
 
 ## What is not achievable
 * Removing an attribute from a framework entity
-* Changing a data type of an entity attribute or association
-* Removing existing entities and features
\ No newline at end of file
+* Changing a data type of an entity attribute
+* Removing existing entities and features
+
+## Successfully implemented features
+* [Shipping method with pickup places](https://github.com/shopsys/demoshop/pull/6)
+    * new shipping method Zasilkovna
+    * pick up places are downloaded by cron
+    * order process change
+    * details in a [issue description](https://github.com/shopsys/demoshop/issues/3)
+* [Product attribute "condition"](https://github.com/shopsys/demoshop/pull/7)
+    * product entity extension
+    * administration form extension
+    * frontend product change
+    * google feed change
+    * detailed info in a [issue description](https://github.com/shopsys/demoshop/issues/4)
+* [Category second description](https://github.com/shopsys/demoshop/pull/8)
+    * category entity extension
+    * administration form extension
+        * new domain attribute
+    * frontend product list change
+    * detailed info in a [issue description](https://github.com/shopsys/demoshop/issues/5)
+* [Twig templates cache](https://github.com/shopsys/demoshop/pull/9)
+    * performance improved by ~15%
+    * cache invalidation is done by 5 minutes interval
+* [Hidden the functionality of the flags](https://github.com/shopsys/demoshop/pull/13)
+    * hidden functionality in administration
+    * hidden functionality in frontend
+    * flags do not affect shop at all
+* [Company account with multiple users](https://github.com/shopsys/demoshop/pull/15)
+    * group user accounts under one company account
+    * separate users login credentials
+    * share users' attributes
+    * change association from 1:1 to 1:N
+        * this change is complicated and potentially dangerous, but achievable
```

Comment:
Do we have a reason why to call this "complicated and potentially dangerous"? It sounds weird to me...